planning/behavior_path_planner/src/scene_module/start_planner/start_planner_module.cpp

```diff
@@ -147,6 +172,12 @@ bool StartPlannerModule::receivedNewRoute() const
          *planner_data_->prev_route_id != planner_data_->route_handler->getRouteUuid();
 }
 
+bool StartPlannerModule::checkCollisionWithDynamicObjects() const
+{
+  return !(parameters_->safety_check_params.enable_safety_check && status_.driving_forward) ||
+         isSafePath();
+}
+
```

Comment:
the following function returns true when a collision is detected.
but `checkCollisionWithDynamicObjects` is the opposite.

```
    if (utils::checkCollisionBetweenFootprintAndObjects(
          local_vehicle_footprint, *backed_pose, stop_objects_in_pull_out_lanes,
          parameters_->collision_check_margin)) {
```

Reply:
changed to `hasCollisionWithDynamicObjects()`